app/containers/App.js

```diff
@@ -61,28 +62,30 @@ class App extends Component {
   })
 
   render() {
-    if (!isLoggedIn(this.props.user)) {
-      chrome.runtime.openOptionsPage();
-
-      return null;
-    }
+    const content =
+      isLoggedIn(this.props.user)
+      ? (
+        <SaveControl
+        article={this.props.article}
+        currentTab={this.props.currentTab}
+        fetchArticleState={this.props.fetchArticleState}
+        rememberArticle={this.props.rememberArticle}
+        rememberArticleState={this.props.rememberArticleState}
+        removeArticle={this.props.removeArticle}
+        removeArticleState={this.props.removeArticleState}
+        updateArticle={this.props.updateArticle}
+        updateArticleState={this.props.updateArticleState}
```

Comment:
I would try props destructuring like so: `<SaveControl {...this.props} />`.

Reply:
`this.props.user` is not passed in.

Reply:
I would pass the `user` then.